templates/zerver/help/name-change.md

```diff
@@ -0,0 +1,16 @@
+#Change your name
+
+![Settings](/static/images/help/settings.png)
+
+If for whatever reason you ever wanted to change your Zuli username, name changes are supported. 
+
+Visit your settings either by clicking the cog widget on the top right corner and then settings or by redirecting your browser to  https://chat.zulip.org/#settings.
+
+Change your name on the Full Name field, and click save changes. 
+
+![Change your name](/static/images/help/name_change.png)
+
+Congratulations! You have a new identity 
```

Comment:
Nitpick: missing a `.` .

I'd also argue that while "new identity" is a fun way to phrase it, we might want to stick with "new name" -- the rest of the user's identity on the server stays the same.